targetFunctions/common/customModelsClass/customModelClass.m

```diff
@@ -50,11 +50,9 @@
 
                 switch thisLanguage
                     case 'matlab'
-                        [thisContrastLayers,allRoughs(i)] = callMatlabCustomFunction(params,i,thisFile,thisPath,bulkIn,bulkOut,numberOfContrasts);
+                        [thisContrastLayers,allRoughs(i)] = callMatlabCustomFunction(params,i,thisFile,bulkIn,bulkOut,numberOfContrasts,0);
                     case 'cpp'
-                        [thisContrastLayers,allRoughs(i)] = callCppFunc(params,bulkIn,bulkOut,i,thisFile,thisFile);
-                    case 'python'
-                        [thisContrastLayers,allRoughs(i)] = pythonCustomFunctionWrapper(thisFile,params,bulkIn,bulkOut,i,numberOfContrasts);
+                        [thisContrastLayers, allRoughs(i)] = callCppFunc(params, bulkIn, bulkOut, i, -1, thisFile);
```

Comment:
If this routine cover both python and C++, may be worth considering renaming it.

Reply:
`callCppFunc` only handles C++ while the `callMatlabCustomFunction` only runs matlab (the wrapper than calls python)